packages/bpk-docs/src/pages/HomePage/HomePage.js

```diff
@@ -74,7 +74,7 @@ class HomePage extends React.Component {
             <BpkGridColumn width={6} tabletWidth={12}>
               <Heading level="h3">Mission</Heading>
               <Paragraph>
-                Bring Design and Engineering together to enable squads to create beautiful, coherent products at scale.
+              To enable design and engineering to build coherent, beautiful and usable products quickly and without pain.
```

Comment:
This line's breaking Travis:

```77:1 error Line 77 exceeds the maximum line length of 120 max-len```

Just split it over two lines and you're good.